src/main/60-eks.tf

```diff
@@ -0,0 +1,71 @@
+resource "aws_iam_policy" "additional" {
+  name = "${var.app_name}-fargate-additional"
+
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action = [
+          "ec2:Describe*",
+        ]
+        Effect   = "Allow"
+        Resource = "*"
+      },
+    ]
+  })
+}
```

Comment:
Do we need this? Check if the EKS module already assigns the correct role+policies

Reply:
Seems like the module create the IAM role (https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/modules/fargate-profile/main.tf)